app/models/concerns/emojifier.rb

```diff
@@ -0,0 +1,14 @@
+module Emojifier
+  def emojify
+    self.content.gsub(/:([\w+-]+):/) do |match|
+      if emoji = Emoji.find_by_alias($1)
+        ActionController::Base.helpers.image_tag(
+          ActionController::Base.helpers.image_path("emoji/#{emoji.image_filename}"),
+          alt: match, class: "emoji"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.